stories/data/geoChart.ts

```diff
@@ -4,6 +4,8 @@ import { IGeoConfig } from "../../src/interfaces/GeoChart";
 import { measure, attribute } from "../../src/helpers/model";
 import { stringToFloat } from "../../src/helpers/utils";
 import { getHeaderItemName } from "../../src/helpers/executionResultHelper";
+// tslint:disable-next-line: no-var-requires
+const fixtures = require("./geoChart/fixtures");
```

Comment:
why don't you use `import`?

Reply:
I'm not sure this case but 'import' get  error:"TypeScript diagnostics (customize using `[jest-config].globals.ts-jest.diagnostics` option)" while run unit test.